src/filepath.c

```diff
@@ -3920,11 +3842,22 @@ unix_expandpath(
     matches = gap->ga_len - start_len;
     if (matches > 0 && !got_int)
 	qsort(((char_u **)gap->ga_data) + start_len, matches,
-						   sizeof(char_u *), pstrcmp);
+			   sizeof(char_u *), pstrcmp);
     return matches;
 }
 #endif
 
+#ifdef MSWIN
+    static int
+mch_expandpath(
+    garray_T	*gap,
+    char_u	*path,
+    int		flags)		// EW_* flags
+{
+    return unix_expandpath(gap, path, 0, flags, FALSE);
+}
+#endif
+
```

Comment:
It seems that the `mch_expandpath()` function defined in `os_unix.c` is identical to this function, but has comments above. Maybe these two can be combined?

Reply:
Good thought.